package com.exception;

public class ArithemeticMain {
	public static void main(String[] args) {
		ArithmeticException1 a=new ArithmeticException1();
		System.out.println("in main");
		a.method4();
		
	}

}
